Adding Documentation. (#8)
hybridx/php.dv: README.md

<h1 align="center">Welcome to php.dv 👋</h1>
<p>
  <img alt="Version" src="https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000" />
  <a href="https://github.com/hybridx/php.dv#readme" target="_blank">
    <img alt="Documentation" src="https://img.shields.io/badge/documentation-yes-brightgreen.svg" />
  </a>
  <a href="https://github.com/hybridx/php.dv/graphs/commit-activity" target="_blank">
    <img alt="Maintenance" src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" />
  </a>
  <a href="https://github.com/hybridx/php.dv/blob/master/LICENSE" target="_blank">
    <img alt="License: ISC" src="https://img.shields.io/github/license/hybridx/php-training" />
  </a>
</p>

> ---------------

PHP Training - Using composer, gulp

##  Requirements

* PHP >= 7.3
* Composer (Tool for dependency management in PHP)

> * Laravel (Web application framework)

Reviewer: ```suggestion
```

* NPM (Node Package Manager)
* Database : MySQL

## Install dependencies
```sh
npm install
```

Laravel
```sh

> composer global require laravel/installer

Reviewer: @XSneha This is not using the laravel installer

```

(To run composer properly you will need OpenSSL extension.Make sure you have openssl extension in php.ini file)
```sh
composer update
```

Create new project with laravel:
```sh

> composer create-project laravel/laravel --prefer-dist

Reviewer: ```suggestion
```

npm install
```

(To run composer properly you will need OpenSSL extension.Make sure you have openssl extension in php.ini file)
```sh
composer update
```

## Run 
* Add a Virtual Host on your local sever with name **_phpdv_** and folder path of your index file.
* Restart DNS if required.

> * Navigate to http://phpdv/ in browser and voilà!

Reviewer: ```suggestion
* Add a Virtual Host on your local sever with name **_php.dv_** and folder path of your index file.
* Restart DNS if required.
* Navigate to http://php.dv/ in browser and voilà!
```
